SADL/AM-CDM-manufacturingProcess.sadl

```diff
@@ -0,0 +1,56 @@
+	described by processSimulation (note "Reference to Simulation records associated with this processStep.") with values of type ProcessSimulation.
+
+ManufacturingSpecification (note "A specification for a manufacturing process") is a ProcessStepSpecification,
+	described by globalProcessParameters (note "Nominal system setting which apply to all material stock involved in the Manufacturing Process Step." ) with value of type GlobalProcessParameters,
+	described by partSpecificProcessParameters (note "Nominal system setting which uniquely apply to an individual part to be processed." ) with values of type PartSpecificProcessParameters,
+ 	described by specifiedEnvironment with a single value of type Environment.
+
+// Suggested 'SpecifiedQuality' with upper and lower bounds
+Environment is a class,
+	described by ambientTemperature with a single value of type SpecifiedQuantity,
+	described by ambientPressure with a single value of type SpecifiedQuantity.
+
+GlobalProcessParameters is a class.
+
+PartSpecificProcessParameters is a class,
+	described by orientation (note "Specified orientation for the specific part." ) with a single value of type Orientation,
+	described by design (note "design file which prescribes the changes to the part which are specific to this process." ) with a single value of type Design.
+
+Orientation is a class,
+	described by partLocationX (note "The x-coordinate position of the geometric center of the design's arbitrarily oriented minimum bounding box with respect to the system origin. ISO/ASTM 52921") with a single value of type Quantity,
+	described by partLocationY (note "The y-coordinate position of the geometric center of the design's arbitrarily oriented minimum bounding box with respect to the system origin. ISO/ASTM 52921") with a single value of type Quantity,
+	described by partLocationZ (note "The z-coordinate position of the geometric center of the design's arbitrarily oriented minimum bounding box with respect to the system origin. ISO/ASTM 52921") with a single value of type Quantity,
+	described by partReorientationA (note "Specification of the rotation of the design's arbitrarily oriented minimum bounding box around the geometric center in the sequence A (rotation about x), B (rotation about y), then C (rotation about z). ISO/ASTM 52921.") with a single value of type Quantity,
+	described by partReorientationB (note "Specification of the rotation of the design's arbitrarily oriented minimum bounding box around the geometric center in the sequence A (rotation about x), B (rotation about y), then C (rotation about z). ISO/ASTM 52921.") with a single value of type Quantity,
+	described by partReorientationC (note "Specification of the rotation of the design's arbitrarily oriented minimum bounding box around the geometric center in the sequence A (rotation about x), B (rotation about y), then C (rotation about z). ISO/ASTM 52921.") with a single value of type Quantity.
+
+Consumables is a class.
```

Comment:
Is a Consumable a type of MaterialInput?

Reply:
I was not consistent in how I implemented this. Thanks for catching that. 

ManufacturingProcessStep has been updated so that materialInputs and consumables are of type MaterialInputs and Consumables. The more descriptive notes for those object properties have been moved to the MaterialInputs and Consumables classes. The classes are not of type MaterialStock, but are simply base classes. They are meant to hold a list of object properties for each consumable (or material input) used in that process. See AM-CDM-build.sadl for a good example.

Reply:
Should be careful when merging pull request #17